runtime/src/lib.rs

```diff
@@ -60,10 +60,10 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: create_runtime_str!("zeitgeist"),
     impl_name: create_runtime_str!("zeitgeist"),
     authoring_version: 1,
-    spec_version: 35,
+    spec_version: 36,
```

Comment:
Why this is required?

Reply:
Is it, because this commit can go straight to production (otherwise it wouldn't get accepted as an updated WASM file)?

Reply:
You can find some info here: https://docs.substrate.io/rustdocs/latest/sp_version/struct.RuntimeVersion.html

Basically, we've changed the transactional interface, so we need to update that version. In particular, we need to update the spec version.

You might be wondering why this wasn't done for the PR where `cancel_pending_market` was removed. I should have incremented the versions there, as well. It doesn't matter though, since this is all part of a single update. I've setup a checklist that we usually use to keep track of these things: https://github.com/zeitgeistpm/zeitgeist/issues/597